resolwe/flow/tests/test_models.py

```diff
@@ -127,6 +127,13 @@ def test_name(self):
         self.assertEqual(second.name, 'User\' data name')
         self.assertTrue(second.named_by_user)
 
+    def test_trim_name(self):
+        process = Process.objects.create(contributor=self.user, data_name='test' * 50)
```

Comment:
Tests shouldn't include too much logic, so I think that it is not necessary to complicate it. If the limit will be changed, `assert(len, 100)` will fail and test will have to be fixed.